test/symbol_name/nmcheck_prefix.pl

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/perl
+
+# Copyright (c) 2017      IBM Corporation. All rights reserved.
+
+# env var MYBASE should be the top dir where ompi/ opal/ orte/ test/ etc live.
+# env var OMPI_LIBMPI_NAME should be @OMPI_LIBMPI_NAME@ from automake
+#
+# Most likely the libs we want to check are
+#     ompi/.libs/lib<mpi>.so
+#     ompi/mpi/fortran/mpif-h/.libs/lib<mpi>_mpifh.so
+#     ompi/mpi/fortran/use-mpi-tkr/.libs/lib<mpi>_usempi.so
+#     orte/.libs/libopen-rte.so
+#     opal/.libs/libopen-pal.so
+# but I hate to assume those are the locations, so I'll use 'find' and
+# test whatever ".so"s are found.
+
+@libs = ();
+$mpi = $ENV{OMPI_LIBMPI_NAME};
```

Comment:
Might want to have a default value for `$mpi` here (or perhaps a warning/abort), just in case someone runs this test outside of `make check`.